website/docs/r/muting_schedule.html.markdown

```diff
@@ -103,6 +128,7 @@ The following arguments are supported:
   - `MutingSchedule`
 - `monitor` - (Optional) The monitors which need to put in the muting schedule. see `monitor_scope_type`:
 - `schedule` - (Required) The schedule information. see `schedule_type`.
+- `notification_groups` -(Optional) The muting schedule group supporting key and values. see `notification_group_type`
 
 #### schedule_type
```

Comment:
nit: same comment as https://github.com/SumoLogic/terraform-provider-sumologic/pull/601/files#r1450949779

Reply:
Hmm, I notice the same thing in other places like `monitor_scope_type`. We should fix them all.

Reply:
updated.